microapps/CalloutApp/app/src/main/java/com/arcgismaps/toolkit/calloutapp/screens/TapLocationScreen.kt

```diff
@@ -170,6 +183,51 @@ fun TapLocationScreen(viewModel: MapViewModel) {
     }
 }
 
+
+/**
+ * Displays the callout at the tapped location on the map or scene.
+ *
+ * This function is responsible for managing callout at the coordinates of the [mapPoint].
+ * It uses animated visibility to show or hide the callout based on the state provided.
+ */
+@Composable
+fun GeoViewScope.GeoViewScopeContent(
+    mapPoint: Point?,
+    calloutVisibleState: MutableTransitionState<Boolean>,
+    rotateOffsetWithGeoView: Boolean,
+    offset: Offset
+) {
+    val lastMapPoint = remember { Ref<Point>() }
+    lastMapPoint.value = mapPoint ?: lastMapPoint.value
```

Comment:
slight improvement here to only update lastMapPoint when `mapPoint` is not null 
```suggestion
    val lastMapPoint = remember { Ref<Point>() }
    if (mapPoint != null) lastMapPoint.value = mapPoint
```

Reply:
actually, i don't think saving the last value is necessary. 
If a null `point` comes in, the callout is going to disappear since the AnimatedVisibility is `calloutVisibleState`, which depends on `mapPoint`